cranelift/filetests/filetests/verifier/jump_table.clif

```diff
@@ -1,19 +1,15 @@
 test verifier
 
 function %br_invalid_default(i32) {
-    jt0 = jump_table [block1, block1]
-
 block0(v0: i32):
-    br_table v0, block2, jt0 ; error: invalid block reference block2
+    br_table v0, block2, [block1, block1] ; error: invalid block reference block2
 block1:
     return
 }
 
 function %br(i32) {
-    jt0 = jump_table [block1, block2] ; error: invalid block reference block2
-
 block0(v0: i32):
-    br_table v0, block1, jt0
+    br_table v0, block1, [block1, block3] ; error: invalid block reference block3
```

Comment:
Kinda weird that this renamed `block2` to `block3`, but since neither of those blocks exist, this still tests the same thing it did before, so that's fine.

Reply:
I did it to differentiate from the previous test, so that it was easier to tell on sight which one was causing errors :D